mappings/net/minecraft/nbt/CompoundTag.mapping

```diff
@@ -60,7 +60,8 @@ CLASS net/minecraft/class_2487 net/minecraft/nbt/CompoundTag
 		ARG 1 key
 	METHOD method_10559 createCrashReport (Ljava/lang/String;Lnet/minecraft/class_4614;Ljava/lang/ClassCastException;)Lnet/minecraft/class_128;
 		ARG 1 key
-	METHOD method_10560 putUuid (Ljava/lang/String;Ljava/util/UUID;)V
+	METHOD method_10560 putUuidOld (Ljava/lang/String;Ljava/util/UUID;)V
```

Comment:
Why "old"? What's the difference between this method and the new one exactly? Also, is it also annotated as `@Deprecated`?

Reply:
The old puts uuid into a sub compound tag with M and L long tags. the new one puts a sub int array tag with 4 uuid components

Reply:
Since the format is different, I think it's a very bad idea to rename the new method to `putUuid`. Any mod using the old version would now be silently switched to use the new method when updating mappings, breaking all old worlds when the mod is updated!

The new method should be named `putUuidNew`, and all the javadocs should make it clear that the format has been changed.

Reply:
or `uuidToSubIntArray`?

Reply:
I think "new" and "old" is good. We can later drop the "new" once the risk of devs not noticing the change is lower.

Reply:
Good idea, suffixed the new methods with `New`. We should probably drop it whenever Mojang removes the old method.